tests/test_self.py

```diff
@@ -1302,6 +1302,21 @@ def test_no_name_in_module(self) -> None:
             [module, "-E"], expected_output="", unexpected_output=unexpected
         )
 
+    def test_nested_min_max_splats(self) -> None:
```

Comment:
I think we can remove this if we also have functional tests(I like functional tests a lot)

Reply:
If I understand correctly there is an issue with `*` in functional tests ?

Reply:
No no, no issue. You can see the functional test .txt file looks great. 

What I mean is that as far as I understand, the functional test will ensure that the message 
nested-min-max appears in a specific line, indicated by adding `# [nested-min-max]` to that line

but what I want to test is not just that the msg is emitted, but that the msg suggestion specifically says "it's possible to do 'max(3, *nums, *lst2)' i". I can only assert this in the unit test, not the functional test.

Reply:
https://github.com/PyCQA/pylint/pull/8234/files#r1100266806 (Modify the expected output in this file and the tests should fail provided you have python 3.8 ore more recent)